infra/examples/aws-google-workspace/main.tf

```
@@ -0,0 +1,186 @@
+terraform {
+  required_providers {
+    # for the infra that will host Psoxy instances
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+
+    # for the API connections to Google Workspace
+    google = {
+      version = ">= 3.74, <= 4.0"
+    }
+  }
+
+  # if you leave this as local, you should backup/commit your TF state files
+  backend "local" {
+  }
+}
+
+# NOTE: you need to provide credentials. usual way to do this is to set env vars:
+#        AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
+# see https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication for more
+# information as well as alternative auth approaches
+provider "aws" {
+  region = var.aws_region
+
+  assume_role {
+    role_arn = var.aws_assume_role_arn
+  }
+  allowed_account_ids = [
+    var.aws_account_id
+  ]
+}
+
+module "psoxy-aws" {
+  source = "../../modules/aws"
+
+  caller_aws_account_id   = var.caller_aws_account_id
+  caller_external_user_id = var.caller_external_user_id
+  aws_account_id          = var.aws_account_id
+
+  providers = {
+    aws = aws
+  }
+}
+
+module "psoxy-package" {
+  source = "../../modules/psoxy-package"
+
+  implementation     = "aws"
+  path_to_psoxy_java = "../../../java"
```

these relative paths usually give trouble. We might think on something, like passing a base path for the installation  and do 
```suggestion
  path_to_psoxy_java = "${var.base_dir}/java"
```